# Monthly Journal Review December 2023

Date: 4 December 2023
Author: Simon Aeschbacher

## Journals covered
- American Journal of Human Genetics (Am J Hum Genet)
- Genetics
- PLoS Biology (PLoS Biol)
- PLoS Genetics (PLoS Genet)
- Proceedings of the National Academy of Sciences of the United States of America (PNAS)
- Theoretical Population Biology (Theor Popul Biol)

## Am J Hum Genet
- [Estimating heritability explained by local ancestry and evaluating stratification bias in admixture mapping from summary statistics](https://doi.org/10.1016/j.ajhg.2023.09.012)  
  heritability, local ancestry, stratification bias, admixture mapping  
  

- [Power of inclusion: Enhancing polygenic prediction with admixed individuals](https://doi.org/10.1016/j.ajhg.2023.09.013)  
  polygenic prediction, admixture, human evolution  
  

## Genetics
- [Scaling the discrete-time Wright-Fisher model to biobank-scale datasets](https://doi.org/10.1093/genetics/iyad168)  
  Wright–Fisher; loss-of-function; selection  
 &#x261E; *Pick*  

- [Genomic mechanisms and consequences of diverse postzygotic barriers between monkeyflower species](https://doi.org/10.1093/genetics/iyad156)  
  Mimulus; Plant Genetics and Genomics; hybrid incompatibility; reciprocal translocation; reproductive isolation; sterility; transmission ratio distortion  
 &#x261E; *Pick* 

## PLoS Biol
- Nothing this month

## PLoS Genet
- [Domain-adaptive neural networks improve supervised machine learning based on simulated population genetic data](https://doi.org/10.1371/journal.pgen.1011032)  
  machine learning, population genetics, domain-adaptive neural networks, simulation mis-specification, domain adaptation, gradient reversal layer, SIA, ReLERNN, dadaSIA  
 &#x261E; *Pick*  
  

## PNAS
- [Sex-dependent phenological responses to climate vary across species' ranges](https://doi.org/10.1073/pnas.2306723120)  
  anemophily; flowering time; intra-specific variation; leaf out; wind-pollination  
  

- [Welcome, AI overlords?](https://doi.org/10.1073/pnas.2318980120)  
  generative AI, scientific writing  
 &#x261E; *Pick*  

- [Emotions and courtship help bonded pairs cooperate, but emotional agents are vulnerable to deceit](https://doi.org/10.1073/pnas.2308911120)  
  cooperation; courtship; deceit; emotion; pair bond  
  

- [Elevational homogenization of mountain parasitoids across six decades](https://doi.org/10.1073/pnas.2308273120)  
  climate change; functional homogenization; insect decline; mountain biodiversity; parasitoid  
  

- [Emergent encoding of dispersal network topologies in spatial metapopulation models](https://doi.org/10.1073/pnas.2311548120)  
  ecological corridors; habitat fragmentation; landscape structure; metapopulation capacity; microscopic models  
  

- [Reimagining infrastructure for a biodiverse future](https://doi.org/10.1073/pnas.2214334120)  
  climate change; conservation; future cities; nature-based solutions; sustainable development  
  

- [Scattered differentiation of unlinked loci across the genome underlines ecological divergence of the selfing grass Brachypodium stacei](https://doi.org/10.1073/pnas.2304848120)  
  Brachypodium stacei; ecological divergence; functional genomics; multiomics; selfing species 
  

## Theor Popul Biol
- Nothing this month


# Summary of Picks

## Spence et al. (2023) Genetics: Scaling the discrete-time Wright–Fisher model to biobank-scale datasets

Abstract:
The **discrete-time Wright–Fisher (DTWF) model** and its **diffusion limit** are central to population genetics. These models can describe the **forward-in-time evolution of allele frequencies** in a population resulting from genetic drift, mutation, and selection. Computing likelihoods under the diffusion process is feasible, but the **diffusion approximation breaks down for large samples or in the presence of strong selection**. Existing methods for computing likelihoods under the DTWF model do not **scale to current exome sequencing sample sizes in the hundreds of thousands**. Here, we present a **scalable algorithm** that approximates the DTWF model with provably bounded error. Our approach relies on **two key observations about the DTWF model**. The first is that **transition probabilities under the model are approximately sparse**. The second is that **transition distributions for similar starting allele frequencies are extremely close as distributions**. Together, these observations enable **approximate matrix–vector multiplication in linear (as opposed to the usual quadratic) time**. We prove similar properties for Hypergeometric distributions, enabling fast computation of likelihoods for subsamples of the population. **We show theoretically and in practice that this approximation is highly accurate and can scale to population sizes in the tens of millions**, paving the way for rigorous **biobank-scale inference**. Finally, we use our results **to estimate the impact of larger samples on estimating selection coefficients for loss-of-function variants**. We find that **increasing sample sizes beyond existing large exome sequencing cohorts** will provide essentially **no additional information except for genes with the most extreme fitness effects**.

## Sotola et al. (2023) Genetics: Genomic mechanisms and consequences of diverse postzygotic barriers between monkeyflower species
The evolution of **genomic incompatibilities causing postzygotic barriers** to hybridization is a key step in species divergence. Incompatibilities take 2 general forms — **structural divergence between chromosomes** leading to severe hybrid sterility in F1 hybrids and **epistatic interactions** between genes causing reduced fitness of hybrid gametes or zygotes (Dobzhansky–Muller incompatibilities). Despite substantial recent progress in understanding the molecular mechanisms and evolutionary origins of both types of incompatibility, **how each behaves across multiple generations of hybridization remains relatively unexplored**. Here, we use **genetic mapping in F2 and recombinant inbred line (RIL) hybrid populations** between the phenotypically divergent but naturally hybridizing monkeyflowers ***Mimulus cardinalis* and *M. parishii*** to characterize the **genetic basis of hybrid incompatibility** and examine its **changing effects over multiple generations of experimental hybridization**. In F2s, we found **severe hybrid pollen inviability** (<50% reduction vs parental genotypes) and **pseudolinkage** caused by a **reciprocal translocation between Chromosomes 6 and 7** in the parental species. RILs retained **excess heterozygosity around the translocation breakpoints**, which caused **substantial pollen inviability when interstitial crossovers had not created compatible heterokaryotypic configurations**. **Strong transmission ratio distortion and interchromosomal linkage disequilibrium** in both F2s and RILs identified a novel **2-locus genic incompatibility causing sex-independent gametophytic (haploid) lethality**. The latter interaction **eliminated 3 of the expected 9 F2 genotypic classes via F1 gamete loss without detectable effects on the pollen number or viability of F2 double heterozygotes**. Along with the mapping of numerous milder incompatibilities, these key findings illuminate the complex genetics of plant hybrid breakdown and are an important step toward understanding the genomic consequences of natural hybridization in this model system.

## Mo & Siepel (2023) PLoS Genetics: Domain-adaptive neural networks improve supervised machine learning based on simulated population genetic data

Abstract:
Investigators have recently introduced powerful methods for population genetic inference that rely on **supervised machine learning** from simulated data. Despite their performance advantages, these **methods can fail when the simulated training data does not adequately resemble data from the real world**. Here, we show that this **“simulation mis-specification”** problem can be framed as a **“domain adaptation”** problem, where a model learned from one data distribution is applied to a dataset drawn from a different distribution. By applying an **established domain-adaptation technique based on a gradient reversal layer (GRL)**, originally introduced for *image classification*, we show that the effects of simulation mis-specification can be substantially mitigated. We focus our analysis on **two state-of-the-art deep-learning population genetic methods** — **SIA**, which infers *positive selection from features of the ancestral recombination graph (ARG)*, and **ReLERNN**, which infers *recombination rates from genotype matrices*. In the case of SIA, the domain adaptive framework also compensates for ARG inference error. Using the domain-adaptive SIA (**dadaSIA**) model, we estimate improved selection coefficients at selected loci in the 1000 Genomes CEU population. We anticipate that domain adaptation will prove to be widely applicable in the growing use of supervised machine learning in population genetics.

Author summary

Population genetic simulation is a powerful tool in the study of evolution. A number of supervised machine learning methods have been developed that take advantage of inexpensive simulations as training data. Despite their outstanding performance in benchmarks, these models can fail when the simulated training data deviate from the real data. In this work, we employed domain adaptation techniques to address this “simulation mis-specification” problem by training the machine learning model jointly with simulated and real data. We performed extensive benchmark experiments to demonstrate the improvement of the domain-adaptive models over standard machine learning models in the presence of different types of mis-specification. In addition, we applied dadaSIA, a domain-adaptive selection inference model, to improve the estimates of selection coefficients at selected loci in a European population. The domain adaptation framework proposed in our work is widely applicable to models relying on synthetic training data and therefore opens the door to many more applications in population genetics.

## Berenbaum (2023) PNAS: Welcome, AI overlords?

[...] “We’re mindful at PNAS that the enabling aspects of technology must comport with cultural values and AI tools must be developed with this constraint in mind. After all, this has been the challenge throughout the evolution of scientific publishing. Although the invention of the movable-type printing press in the 15th century was critical to fostering the Scientific Revolution, even mechanized printing had its detractors. Some authors were reluctant to embrace this new technology out of fear that the newfangled mechanical devices would introduce errors for which they as authors would be responsible, at the price of their reputation (11). ChatGPT doesn’t care about its reputation, but twenty-first century authors should take a page, as it were, from their fifteenth century predecessors and recognize it’s their reputation on the line when they use these newfangled AI tools.” 