---
title: Review of Manuscript GENETICS-2024-307195 by Sakamoto *et al.* “The persistence of locally adapted polymorphisms under mutation swamping”
author: Simon Aeschbacher
date: 20 August 2024
geometry: "left=3cm,right=3cm,top=2.5cm,bottom=2.5cm"
output: pdf_document
linestretch: 1.15
---

## Summary
Sakamoto and co-authors use analytical theory and simulations to investigate the fate of a focal, locally beneficial allele of large effect as a function of the overall mutation rate at an infinitesimal number of small-effect loci in the background. All loci contribute to a quantitative trait under spatially divergent stabilising selection. The authors' undertaking is of high interest because it contributes to our understanding of why and when fundamentally different types of genetic architectures of quantitative traits evolve under divergent selection with gene flow. Methodologically, the authors make a significant contribution to linking population genetic with quantitative genetic models in the context of a trait under locally divergent selection for different optima. This endeavour is challenging because of the different natures of the respective models and the disjoint, if not opposing, sets of assumptions about the respective sets of genetic loci. The authors tackle this challenge by decoupling the two models as much as possible at the genetic level (there is free recombination between the focal locus and the background loci), while keeping them connected through the joint dependence on the equilibrium mean trait of the focal allele frequency and the mean genotypic value of the background regions. This is a clever approach that yields results that increase our understanding of how different evolutionary forces interact to cause and modulate the evolution and maintenance of local adaptation. I am very positive about the manuscript and have no doubt that the overall quality and scope of the analyses and results justify its publication in GENETICS. The paper is well written. I liked how the authors embed the study into the context of the existing literature. However, I have a few major concerns regarding the authors' agenda of promoting “mutation swamping” as a description of what they found, the robustness of the results to the assumption of linkage equilibrium, and the metric the authors use to classify their simulations into those leading to a stable vs. transient architecture. I also had a substantial number of Minor Comments regarding language. Overall, I am positive that these minor issues can be fixed fast. The major issues will take some effort to fix and/or address, but I am confident that the manuscript can ultimately be published in GENETICS after a thorough revision.

## Major Comments
1. I am concerned about the coining and use of the term “mutation swamping”, which the authors justify based on a supposed analogy to “migration swamping”. I am afraid this analogy is superficial at least, if not conceptually wrong and thus misleading.  I am against using “mutation swamping” in the context of the author's manuscript for the following reasons:
    1. I can see that the replacement of what the authors call a “stable divergent [genetic] architecture” by a “transient architecture” at high background mutation rates is analogous as an outcome to what Yeaman and Whitlock (2011) observed at intermediate migration rates. However, as far as I can see, “swamp”-ing by high levels of gene flow was used in Yeaman and Whitlock (2011) to describe the *loss of divergence at the weakly selected alleles*, while *divergence is maintained at one or a few strong loci*. In contrast, the authors of the current manuscript are concerned with the *loss* of divergence at the *focal* locus in favour of what they call a “transient” architecture under which divergence is maintained by ephemeral weak-effect alleles at many background loci: “However, as the mutation rate increases, divergence at large effect loci becomes unstable, and the architecture transitions to a transient one driven by fleeting divergence at many small effect loci dispersed across the genome. We termed this collapse of the stable divergent architecture ‘mutation swamping’, with an analogy to the classic migration swamping (Haldane 1930; Wright 1931; Felsenstein 1976; Lenormand 2002).” Thus, the analogy between migration swamping *sensu* Yeaman and Whitlock (2011) and “mutation swamping” as used in the current manuscript remains fuzzy at the level of an evolutionary pattern.
    2. Related to the previous argument, but more fundamentally, the term “mutation swamping” in analogy to migration swamping is unjustified in the current context at the genetic and mechanistic level. In the classic literature also cited by the authors (Haldane, 1930; Wright, 1931; Felsenstein, 1976), migration swamping describes the removal of a locally beneficial allele at a focal locus by an immigrant allele at the same locus coming in through gene flow via migration. This is both a _directed_ process – migration is directional and running counter to selection on average unless in special configuration – as well as a _direct_ process in two ways: it is direct in the sense that migration is the force immediately acting on and causing the loss of the locally beneficial allele, and direct in the sense that the process acts on the focal locus. In contrast, what the authors term “mutation swamping” neither is the result of a directed process nor does it act directly on the focal locus where divergence is lost. Instead, mutation as modelled in the quantitative framework that the authors use for the background loci is not directed: effect sizes follow a truncated Gaussian distribution. Moreover, the mutational process that leads to what the authors term “mutation swamping” (i.e., the loss of divergence at the focal locus) does not act directly on the focal locus – there *is* no mutation at this locus. From the perspective of the focal locus, mutation at the background loci merely has a *modulating* effect on the joint action of selection and migration at the focal locus. In my opinion, there is just too much conceptual incongruence for there to be any justification in using the term “mutation swamping”. In this context, the authors also imprecisely cite Höllinger et al. (2019, 2023) on lines 16 to 19: Höllinger et al. did not show that mutation rate *drives* selective sweeps nor small allele-frequency shifts at many loci. These authors only showed that mutation rate *modulates* the way by which polygenic adaptation occurs. The *driving* force is and remains selection, both in the sweep-like and in the shift-like scenarios of oligo- or polygenetic adaptation. In my view, this imprecise citation reflects the conflation of direct and indirect effects of mutation that the authors seem to be willing to accept in using “mutation swamping“ in the given context. I think this conflation is misleading and must be avoided.
    3. My concern was reinforced when I read lines 43 to 53 of the current manuscript: “The range of migration rates within which mutation swamping occurs shifts toward lower values as the effect size at the focal locus decreases (Figure 4D). This pattern suggests that *mutation swamping will also re-shape the architecture of adaptation through homogenizing small effect alleles while preserving divergence at large effect loci. This effect is analogous to the evolution of a concentrated architecture by migration swamping* [emphasis added] (Yeaman and Whitlock 2011; Yeaman 2015; Rafajlović et al. 2016) [...]”. The authors introduce and repeatedly refer to “mutation swamping” as the *loss* of divergence at the focal locus in favour of a transient architecture involving many background alleles. Yet, here the authors seem to imply that “mutation swamping” could lead to exactly the opposite, i.e., an architecture with a large-effect locus at which there is divergence, and no divergence at the background loci. This passage thus read to me as if it inverted the authors' own definition of “mutation swamping” (see above), and so would also break with the analogy to what happens at the focal locus under migration swamping under classic theory (e.g., the works by Haldane, 1930; Wright, 1931; Felsenstein, 1976 cited earlier by the authors). I thus felt the argument became not only inconsistent, but also confusing.
2. For the sake of analytical tractability, the authors assume free recombination between the focal locus and the background loci. I think this assumption is justified as far as the analytical work goes for the current manuscript. However, the assumption could have non-negligible consequences on the generality of the results. The work by Yeaman and Whitlock (2011), follow-up papers by Yeaman and co-authors, as well as work by others collectively have shown that physical linkage between a major locally divergent locus and nearby minor-effect loci has a qualitative effect on the dynamics and outcome of evolution at the genetic level. Such linkage therefore also affects the extent to which alternative genetic architectures of polygenic traits under spatially divergent selection evolve and are maintained. In fact, the authors report that in some of their simulations, “a large effect locus arises in the background region by tight linkage of small alleles and takes over the role of the cluster.” Although it remains unclear what proportion of simulations resulted in this outcome, it is clear that linkage is of qualitative importance. Given that the focal locus starts out from perfect adaptation in the simulations, whatever the proportion of simulations of interest was, it would be a lower estimate of the propensity of clustering to evolve around the focal locus. Such clustering is expected to increase the persistence of the concentrated architecture around the focal locus, and thus shift the odds in favour of the concentrated architecture. For this reason, I think the authors should aim to report simulation results that include some degree of linkage between the focal and the background loci. If this is too cumbersome, I think the authors should at least thoroughly discuss the limitations that could arise from the assumption of free recombination.
4. Measure of stability $T^{\ast}$.
5. ...

## Minor Comments
**C:** comment; **Q:** question; **S:** suggestion; **R:** request.

### Title
- **C:** The term persistence sounds confusing (contradictory) in relation to the high-mutation-rate regime that ‘mutation swamping’ implies. Essentially, the paper shows that locally adapted polymorphisms tend *not* to persist under high mutation rates. ***S:**: Replace by ‘fate’.
- **C/R:** As explained above (Major Comment 1), I find the term “mutation swamping” misleading und not justified. I request its omission from the paper and the title. ***S:**: Replace by “high mutation rates”. In combination with my previous comment, I suggest the title be modified to “The fate of locally adapted polymorphisms under high mutation rates”. **R:** Please also adjust the running title “Mutation swamping in local adaptation” so that it does not contain “mutation swamping”.

### Abstract
- [l.2–3] **C:** The two descriptors “*pronounced divergence *[at a few loci]” and “*small allele frequency shifts* [at many loci]” do in my view not not describe two genetic architectures, but a mechanistic/procedural aspect not typically considered to be part of the genetic architecture, but in this case causally linked to the coming-into-place of architectures. **S:** Insert “and evolutionary histories” after “architectures“ to solve the problem.
- [l.3] **R:** Insert “the” before “migration rate”.
- [l.4] **C/S:** I find “stable contributions” a fuzzy term and suggest “lasting” instead.
- [l.6] **R:** As explained above (Major Comment 1), I request that the authors omit the term “mutation swamping” because it is an inappropriate analogy to “migration swamping” in the context of this paper. **S:** Replace “... ‘mutation swamping’ phenomenon depends upon ...” by “... transition in genetic architecture depends on ...”.
- [l.7] **S:** Replace “determining mutation effects” by “describing the mutation process”. **R:** Please use present tense for “develop” to clarify that the theory is developed in your paper.
- [l.7–9] **C/S:** I find this sentence too long and complicated. I suggest to split after “occurs” and to then start with “We find that ...”.
- [l.8] **R:** Replace “swamping” by “the transition”. Insert “a” before “wide range” and “the” before “parameter space”.
- [l.9] **R:** Replace “mutation swamping threshold” by “threshold mutation rate”.
- [l.10] **S:** Omit “and” before “weakly”; replace “but is only minimally affected by” by “and marginally on the population size” to streamline the phrasing. **R:** Insert “the” before “parameter space”.
- [l.11] **R:** Replace “mutation swamping” by “the transition to a transient architecture”.


### Introduction
#### Page 1
- [l.46] **C/S:** The term “linked selection” has become widely used, but this does not change my opinion that it is imprecise. I suggest to replace it by “selection at linked sites”.
#### Page 2
- [l.2, l.6, and other places] **C/S:** To increase clarity, I suggest to replace “small effect” and “large effect” by “small-effect” and “large-effect” here and throughout the paper whenever these pairs of words occur as compound adjectives.
- [l.5–7] **R:** Please back this sentence with (a) reference(s) or rephrase it to be more tentative.
- [l.9] **S:** Omit “rate” after “mutation” or insert “the” before “mutation rate”.
- [l.17] **S:** Replace “maintain divergence” by “persist”.
- [l.18–20] **C:** I missed an indication after “causal alleles” of the number of loci involved – more than in the case of the constant architecture, right? **S:** Insert “at many loci” after “causal alleles”. **C/S:** Replace “balance” by “under the joint action” because “balance” is problematic if understood as applying to the level of individual loci.
- [l.26–31] **S:** Replace “transition into” by “become”. **C/S:** I find “as various parameters change” a bit bold given the moderate number of parameters explored. I suggest to write “... threshold change as a function of the effective population size, selection strength, and the effect size of alleles under selection, and (iii) why ...”. To me, this phrasing is more accurate and thus gives a better idea of the scope of the study. I suggest to replace the commas separating the three objectives by semi colons, just to make things more clear.
- [l.48] **S:** Replace “... consider their interaction, we ...” by “... consider the interaction of these two types of loci, we ...” to avoid ambiguity.
- [l.50–54] **R:** Correct “combines” to “combined” and replace “one locus two allele model” by “one-locus two-allele model” to increase clarity.
- [l.64] **C:** “Show“ would do instead of “demonstrate“, I think.
- [l.69–72] **C/R:** As explained in Major Comment 1, the similarity of the process studied here is to  “migration swamping” is too superficial to justify an analogy and I therefore strongly argue against coining the term “mutation swamping” in this context. I request that this sentence be omitted.



### Results
- [l.128] **Q:** Did you use ADMIXTURE (as stated in the Methods) or STRUCTURE (as stated here)?
- [l.173] **Q:** Is there a good reason for why the authors performed the functional enrichment analysis on the set of 3,539 highly CN-differentiated genes, and not the 2,192 very highly CN-differentiated genes?
- [l.237–239] **Q:** I wonder about the interpretation of the result here, i.e. about the speculation that a high CN expansion rate along an internal branch in the population tree could be driven by a high rate of expansion along one of the subtending terminal branches. Does CAFE v4.2.1 not separate the rates on internal branches from the rates on terminal branches? If it does, how can the rate along the internal branch be inflated because of a high rate along just one of the subtending terminal branches (leaves)?
- [l.253] **S:** I wonder if it would be better to break the paragraph here, but to omit the break in l.247.
- [l.272] **S:** I suggest a paragraph break here.
- [l.329–330] **C/S:** Fig. 5B suggests that Bio7 and ann_Rmean are correlated, and so it may not be too surprising that the sets of genes associated with these two variables overlap. I think you should mention this point.
- [l.340-342] **C/S:** It was not clear to me what test the result stated here was based on. Could you please clarify in the text? An analogous comment applies to the statement in l. 343–344.
- [l.344–349] **C/R:** This part is on the side of a Discussion paragraph, with quite some interpretation and speculation mixed into what should be a summary of the results. I think the authors should use a more neutral wording for points (ii) and (iii), i.e. language that is more descriptive of the results, and less on the side of interpreting these results as showing certain evidence of locally adaptive divergence.

### Discussion
- Generic comment to the Discussion: Please more clearly differentiate between the scope of the study and its study system vs. the broader context. Specify what taxa were the subject of studies you cite.
- [l.351–353] **C:** The scope of this statement remains unclear. Does the sentence refer to plants in general, to wild tomatoes, to *S. chilense*? I also think the authors should start the Discussion with a concise reminder of the purpose of the study.
- [l.369–370] **C/S:** I found this phrasing overloaded. How about: “We identified patterns of gene CN variation that likely represent footprints of adaptive divergence.”
- [l.381–383] **R:** Please specify the taxa for which the result reviewed here was found. The references given do not seem to be specific to *S. chilense*, but to include other taxa.
- [l.399–402] **C:** I wonder if it is necessary to mention the anthocyanin pathway in cultivated tomato in the way it is. Either omit this statement or make clear how it relates to your finding.
- [l.408–410] **C/R:** This sentence illustrates the problem with the generic use of present tense: The statement made here sounds like a general statement because you use present tense. But the statement is meant to be specific to the study. Please rephrase to use past tense and to make clear that you are referring to your own result in this study.
- [l.424] **C/R:** This is the first time it is stated that these genes are chloroplast genes found in the nuclear genome, so the reader may need more than just a clause to appreciate the fact. Please expand to increase clarity.
- [l.436–438] **Q/S:** Do you mean that the tests with simulations showed that your approach was likely conservative? If so, please rephrase to make this point more clear.  Also, it would make sense to repeat that the simulations simulated short-read data.
- [l.440–441] **C/S:** How does this sentence relate to the study? I do not classify GEA as a selective sweep method. I also do not think the authors used ‘several’ methods, but just RDA coupled with LFMM. Please rephrase to a more precise statement.
- [l.444–447] **S:** Please expand a bit on how, not only that, the new method will help.

### Materials and Methods
- Generic comment: There is currently a considerable degree of redundancy between the Methods in the main text and those in the Supplementary Text. When revising the manuscript, I suggest that you make an effort to reduce this redundancy.
- [l.483] **Q:** I did not understand why there is a factor of two in the formula for the copy number. Could you please state that in the text?
- [l.485] **R:** Please explain what $V_{\mathrm{ST}}$ is, or refer to the Supplementary Text.
- [l.499–500] **Q:** Please justify why you forced the tree to be ultrametric. Is it justified to assume a constant molecular clock? Does the rate of change of CN depend on $N_e$ or does $N_e$ cancel? I ask because branches in TreeMix trees scale with $N_e$, and if one forces the tree to be ultrametric, one loses the scaling by $N_e$.
- [l.511] **C/S:** I felt that a transition was missing between the previous sentence and this sentence here (“LFMM ...”). Please insert one.
- [l.513] **Q/R:** Did you implement LFMM2? If so, please provide a reference to the code.

### Figures
- Generic comment: I find it difficult to read and interpret bar plots in which bars belonging to different categories are stacked (e.g. Fig. 1B, E; Figure S6A; Figure S13B, C). Did the authors consider alternatives, e.g. clustering bars side by side, or plotting individual bar plots for each category on the same horizontal line so that it is easier to compare values for a given category across different classes on the x-axis?
- Fig. 1:
    - **R:** The map in panel A is too small. Consider enlarging that panel and reorganising the other panels. **S:** Panel B seems fully redundant with Table S1, so I think the authors could drop Table S1.
    - [l.781–782] **R:** There seems to be a confusion between accessions and individuals. As far as I understood, population is equivalent to accession in this case, but what is “accessions” here should read “individuals”.
- Fig. 2:
    - **R:** The colour scheme in the ADMIXTURE plot does not seem to correspond well with the colour scheme of the PCA. For instance, for K = 7, it is confusing to see the purple plots below the blue SC_LA4107. Also, the shades of the respective colours do not match (e.g., the green in the ADMIXTURE plot seems too bright compared to the green of SC_LA2932; neither the light green nor the dark yellow in the ADMIXTURE plot seem to unambiguously match the olive green of C_LA2931. Please adjust the colour scheme of the ADMIXTURE plot so that it matches the one of the PCA in panel A.
    - [l.788] **R:** There is apparently again a confusion between accessions and individuals. According to the Introduction, there are 35 individuals from seven populations (accessions), not 35 accessions.
    - [l.788] **Q:** Did you use ADMIXTURE or STRUCTURE (it says “Structure analysis” here, but in l.474 you state you used ADMIXTURE)?
- Fig. 4:
    - **C/S:** The legend in panel B was unclear to me. Specifically, what is meant by “proportion of CN [copy number] lost” and “proportion of CN gained”? Do you mean “proportion of genes with CN loss” and “proportion of genes with CN gain”, respectively? Please rephrase.
    - [l.806–813] **S:** In the caption, I suggest to improve the wording when you write about gene copy number gain and loss (see specific suggestions in the annotated PDF).
- Fig. 5:
    - [l.821–822] **R:** The title sentence to me sounds to interpretative. I think the authors should chose more descriptive, neutral wording.
    - [l.824] **Q:** Could you please check if this is precisely the meaning of the vector lengths? Is it not that the projection of a vector onto an ordination axis shows the correlation with that axis? Please rephrase if necessary.

### Tables
- Table 1: See minor fixes in the annotated PDF.

### Supplementary Text
- Please refer to the annotated PDF. I did not make detailed suggestions for the entire text because I was a bit overwhelmed by the density of writing and language issues. Please revise the entire text carefully before submitting a revision.

### Supplementary Figures and Tables
- Generic comment: The captions of the supplementary figures and tables seem incomplete and rudimentary at times. Please add full title sentences and also make full sentences in the remainder of the captions.
- Figure S2:
    - There seems to be again a confusion between “accession” and “individual”. In the label of the x-axis, it should say “individuals“ in my view.
- Figure S3:
    - **R:** See my comment to Fig. 2 w.r.t. the colour scheme of the ADMIXTURE plot in panel C (it does not match the colours assigned to the accessions and used in panel A).
- Figure S5:
    - **Q/R:** Do the dots in the figure represent genomic windows or genes? Please adjust the caption if necessary (it currently mentions “genes”).
- Figure S6:
    - What is meant by “CN value(s)”? Please define / write out once at least.
- Figure S7:
    - **C/R**: The description in the caption of panel C is unclear. Do you mean Do you mean “The number of genes associated with a response to ...”? Please fix.
- Figure S9:
    - **Q/R:** Am I right in thinking that this figure is reproduced from Wei et al. (2023a)? If so, I do not think it is necessary (nor appropriate) to reproduce the figure here.
- Figure S11:
    - **C/R:** The caption misses a title sentence. Please fix.
    - **R:** The second part of the caption needs revision. The wording used to describe the RDA plot is unclear. See detailed comments in the annotated PDF.
- Figure S12:
    - **R:** Please add a title sentence to the caption and describe what panel A shows.
- Figure S13:
    - **R:** Please revise the title sentence to be more informative.
- Table S3:
    - **C:** This table seems fully redundant to Figure S2 and could in my view be omitted.
- Table S4:
    - **C:** This table seems fully redundant to Fig. 1E and could in my view be omitted.
- Table S5:
    - **C:** I do not think the footnote is necessary given the column labels in the table and the details given in the Methods and Supplementary Text.