---
title: Review of Manuscript MBE-23-0129.R1 by Thawwornwattana *et al.* "Inferring the direction of introgression using genomic sequence data"
author: Simon Aeschbacher
date: 27 April 2023
geometry: "left=3cm,right=3cm,top=2.5cm,bottom=2.5cm"
output: pdf_document
header-includes: 
- \usepackage[document]{ragged2e}
linestretch: 1.15
---

## Summary


## Major concerns

1. ...
2. The acronyms used by the authors to denote the demographic scenarios are confusing and inconsistent with previous literature that introduced the scenarios. Specifically, it is unclear to me why the "MSC-with-introgression" model should be abbreviated as "MSci", given that "MSC" (and not "MSc") is the abbreviation for "Multi Species Coalescent" apparently recognised by the authors. Second, Yu et al. (2012) already introduced the acronym "MSNC" for the exact same model as the one the authors consider here. In short, I recommend that the authors use "MSNC" instead of "MSci" for better congruence with the previous literature and to aim for a parsimonious use of terms in the literature.
3. I am concerned about the length of manuscript, especially the Results section. Large parts on pages **X to Y** elaborate on results from the three- and four-population models that are analogous to the respective results obtained under the two-population model. I suggest that a detailed account of the results for the three- and four-population model be deferred to a Supplementary Text. The main text could contain just a brief summary of the overall correspondence between the results obtained under the models with different numbers of populations. Together, the the results from the three- and four-population model in my view should not absorb more than a single page in the main text.
4. In the application to the *Heliconius* data, the differences between coding and noncoding DNA in the estimates of the mutation-scaled split times and population sizes are striking (see Figure S7). The difference is particularly large for the time to the root, $\tau_r$, with estimates for noncoding DNA generally at least two-fold higher than estimates for coding DNA. The authors explain this difference by a reduced (effective) neutral mutation rate in coding regions due to purifying selection. Can the authors verify this explanation by comparing the differences in respective parameters between coding and noncoding DNA found in previous studies of the same two focal *Heliconius* species?
5. In the application to the *Heliconius* data, and as the authors briefly mention (p. 11, l.11–15), estimates of some parameters (including $\theta_C$, $\theta_M$, $\tau_s$, and $\tau_c$) appear as outliers for a subset of chromosomes (chromosomes 5, 10, 13, 15, 19, and/or 20, depending on the parameter). It also seems as if these estimates are in most cases stronger outliers for the noncoding data as compared for the respective coding data. I found it particularly striking that $\tau_c$ . I missed an attempt at explaining this variation, and an effort to check if the outliers might be due to a technical artefact or if they reflect a biological signal.

## Minor comments
**C:** comment; **Q:** question; **S:** suggestion; **R:** request.

### Abstract
- No comments.

### Introduction
- [p.1L, l.39–43] **R:** The Introduction starts with a view of gene flow that is biased towards its facilitating role in speciation. The emphasis of gene flow as a facilitator of diversification and adaptation is a comparatively recent one, and it is much less well supported than the longer established view of gene flow as hindering diversification and (local) adaptation. In short, I encourage the authors to rephrase "Gene flow between species is an important evolutionary process that can facilitate species diversification and adaptation" to "Gene flow between species is an important evolutionary process that can both facilitate [appropriate references] as well as limit [appropriate references] species diversification and adaptation".
- [p.1L, l.43] **R:** "It occurs ..." &rarr; "Gene flow occurs ..."
- [p.1, l.48–49] **Q:** Do all three references refer to all three factors (mate choice, ecological selection, hybrid incompatibility)? If not, please distribute the references accordingly.
- [p.1L, l.52–53] **S:** Insert comma after "species" and replace ", being ..." &rarr; ", i.e. ..."
- [p.1R, l.49] **S:** Omit "or $\gamma$" to avoid confusion.
- [p.1R, l.42–47] **R:** Use "MSNC" instead of "MSci" throughout the manuscript, and simplify the sentence here accordingly.
- [p.1R, l.50–54] **R:** Use "IM model" instead of "MSC-M model" throughout the manuscript,  do not introduce "MSC-M" here, and simplify the sentence here accordingly.
- [p.2L, l.7] **S:** Specify the reference to Westram et al. (2022) as a reference to a review, e.g. "(see Westram et al. 2022 for a review). Otherwise, cite original papers on the concept of effective gene flow, e.g. Barton & Bengtsson (1986) Heredity, or Petry (1983) Theor. Popul. Biol.
- [p.2L, l.18] **R:** Insert ", at least in principle, " after "... and thus", to defuse the contradiction between the statement that multilocus sequence alignments are informative about the direction of gene flow on the one hand, and the counter example given just below (lines 21–27).
- [p.2L, l.28–29] **S:** Please back this sentence with a reference.
- [p.2L, l.30–33] **S:** The authors may want to cite Hibbins & Hahn (2019) Genetics, doi: 10.1534/genetics.118.301831, and Hibbins & Hahn (2021) Genetics, doi: https://doi.org/10.1093/genetics/iyab220.
- [p.2L, l.33–34] **C:** The transition between these two paragraphs, and thus from describing background to stating the questions addressed in this paper, seems to abrupt. I suggest to work out the gap of knowledge addressed by the paper somewhat more precisely. At least, the authors should *first* mention that they will now state the questions addressed in the paper, and then state the questions – not the other way round as they currently do (cf. l.48–49).
- [p.2L, l.34] **S:** Please introduce the notation "A $\rightarrow$ B" at this point.
- [p.2L, l.50–51] **S:** Omit the parentheses surrounding "in particular, the introgression probability", and insert commas after "... of parameters" and after "(Flouri et al. 2020)".
- [p.2L, l.52] **R:** Insert "a" before "Bayesian".
- [p.2L, l.54] **S:** Omit "Bayesian".
- [p.2L, l.56] **R:** Insert "the" before "$D$ statistic".
- [p.2L, l.55 to p.2R, p.10] **C/S:** I find the switch here back to providing more background and more on the gap of knowledge addressed in the paper after the statement of the questions unfortunate. I thus suggest to place this content before the statement of the questions.
- [p.2R, l.7–10] **R:** This sentence is incomplete; please formulate a full sentence.
- [p.2R, l.17–22] **C:** It seems as if the authors here provide a second list of objectives on top of the objectives (questions) stated earlier on (p.2L, l.35–48). This apparent second set of objectives is confusing. I suggest the authors be more clear about the core objectives of the paper (inference of the extent, timing, and direction of gene flow), and that they differentiate the core objectives from lower-level objectives (number of species included in the analysis, inflow from sister vs. non-sister species).
- [p.2R, l.29–31] **C:** The claim "to demonstrate the feasibility of inferring the direction of gene flow, as well as its timing and strength" by application to empirical data in the absence of information about the ground truth is problematic. I think this is not feasible. I suggest to rephrase the sentence so that the purpose of the application to the *Heliconius* data becomes a bit more modest (realistic), e.g. "..., and to demonstrate how the framework can be applied to inferring the direction of gene flow, as well as its timing and strength.".



### Results
- [p.2R, l.47–50] **S:** To increase clarity: "Divergence time is defined as $\tau = T \mu$, where $T$ is the divergence time in generations and $\mu$ is the mutation rate per site per generation." &rarr; "Throughout, we measure time in multiples of the expected waiting time to a mutation, and thus define the scaled divergence time $\tau$ as $\tau = T\mu$, where $T$ is the divergence time in generations and $\mu$ is the mutation rate per site per generation."
- [p.3L, l.9] **R:** I find $N$ an unfortunate choice for the number of sites, as it is easily confused with the much more common meaning of population size. Please use another symbol, e.g. "S", instead of $N$.
- [p.3L, l.13] **S:** I think it would be unwise to coin this assumption as the standard. Thus, please replace "We make the standard assumption of ..." &rarr; "We assume ...".
- [p.3L, l.30–31] **S:** Insert "... and constant population sizes" after "... no gene flow".
- TODO(Simon): Go on here.

### Discussion
- [l.1088] **S:** ...


### Methods
- [l.1539–1545] **C/S:** ...


### Appendix A
[l.1915–1924] **Q:** Were the likelihood functions derived here not already provided collectively by Wilkinson-Herbots (2008 Theor Popul Biol; 2021 Theor Popul Biol) and Costa and Wilkinson-Herbots (2017 Genetics; 2021 Theor Popul Biol)? **R:** Please make clear which results were obtained before and by whom, and which results are novel.

### Figures and Tables
- [Figure S4]
    - **S:** In the caption, replace the last sentence "See legend to figure 6." by "Other details as in Figure 6.".
- [Figure S6]
    - **S:** In the caption, replace the last sentence "See legend to figure S5." by "Other details as in Figure S5.". 