---
title: Review of Manuscript MBE-24-1101 by Wei *et al.* â€œCopy number variations shape genomic structural diversity assiciated with ecological adaptation in the wild tomato *Solanum chilense*â€
author: Simon Aeschbacher
date: 22 November 2024
geometry: "left=3cm,right=3cm,top=2.5cm,bottom=2.5cm"
output: pdf_document
linestretch: 1.15
---

## Summary
...

## Major Comments
1. ...
2. I was not able to access the script files from the GitLab repository at the LRZ center for computation. I therefore could not (spot) check the code. I consider this a limitation that should be fixed in the process of peer review if the journal wants the review to cover the supplementary code files. Multiple URLs given to specific sections within the GitLab repository are given throughout the Methods section, but none of them leads to content that is publicly accessible.

## Minor Comments
**C:** comment; **Q:** question; **S:** suggestion; **R:** request.

### Title
- **C:** I appreciate the modification from â€œunderpinningâ€ to â€œassociatedâ€.

### Abstract
- ...

### Introduction
- [l.pâ€“p] **C:** 

### Results
- [l.152â€“153] **R:** Replace â€œIn Fig. S6, we showed ...â€ by â€œIn Fig. S6, we show ...â€ (present tense), or, better, write â€œFigure S6 shows ...â€

### Discussion
- [l.pâ€“p] **C:**
- In their response to comment no. 6 by Reviewer 1, the authors state that they now added a few lines to the last paragraph of the Discussion about the shortcomings shared between $F_{\mathrm{ST}}$ and $V_{\mathrm{ST}}$. Reviewer 1 was concerned with $V_{\mathrm{ST}}$ being inflated due to low within-population variation in CNV. **TODO(Simon): Go on here. Finish the point.**

### Materials and Methods
- [l.492â€“492] **C:** The abbreviation â€œSVâ€ for â€œstructural variationâ€ has already been introduced in the main text. The repeated definition here likely results from the fact that the authors moved some methods text from the Supporting Materials to the main text.
- [l.493â€“495] **S:** Perhaps rephrase to â€œThis evaluation enumerated ... and/or duplications.â€ to â€œThis study found that combining SV detection tools tends to give higher precision and recall than individual tools and that LUMPY (Layer, et al. 2014), Manta (Chen, et al. 2016), Wham (Kronenberg, et al. 2015) and DELLY (Rausch, et al. 2012) are tools with good overall performance for deletions and duplications.â€
- [l.495â€“497] **C:** This sentence does not seem to be complete. It remains unclear what the difference is between â€œtoolâ€ and â€œalgorithmâ€. It seems as if the authors use â€œalgorithmâ€ for what Kosugi et al. (2019) used â€œmethodâ€, whereas Kosugi et al. (2019) used â€œalgorithmâ€ for the software tool. **S:** I suggest that the authors restrict their statement here to what is strictly necessary for the reader to obtain an overview of what information from the sequencing data data was used to call CV overall. Later in the text, when the application of specific tools is described, the description can state what information the respective tool relies on. Specific suggestion for here: â€œThese tools implement different calling algorithm that jointly draw information from patterns in read pairs, split reads, read depth, and de novo assemblyâ€.
- [l.498] **S:** For consistency with l. 493, I suggest to write â€œLUMPYâ€ in capital letters. This comment also applies to other occurrences of the tool name.
- [l.498â€“500] **S:** Rephrase â€œ, and the split-read alignments also were extracted ...â€ to â€œ, and then we extracted the split-read alignments ...â€ to increase consistency (active voice) and clarity (repetition of the phrasing reassures to the reader that both steps are done to extract data to be analysed, not to be excluded).
- [l.502] **S:** Rephrase â€œCNV calling used DELLY ... into vcf file (Danecek, et al. 2011; Danecek, et al. 2021)â€ to â€œFor CNV calling with DELLY v0.7.6, we chose the default parameters and converted the output file from bcf to vcf format using bcftools v1.9 (Danecek, et al. 2011; Danecek, et al. 2021)â€.
- [l.504] **S:** Omit â€œFurthermoreâ€ and just say â€œWe also ran ...â€.
- [l.506â€“507] **S:** Replace â€œas 50bpâ€ and â€œas 1Mbâ€ by â€œto 50 bpâ€ and â€œto 1 Mbâ€, respectively. Not that I suggest to insert spaces between the numbers and the units to be consistent with other places in the manuscript, e.g., l. 541 to l. 543. **C:** It was unclear to me what is meant by â€œ..., and CNV types and DNA strands must matchâ€œ. Please clarify and improve the formulation.
- [l.509] **R:** The current phrasing does not work. Please rephrase to â€œWe (finally) used the merged CNV set **as input** for SVTyper v0.7.0 to ...â€.
- [l.509â€“512] **S:** I think this part could be written more compactly as â€œWe used the merged CNV set as input for SVTyper v0.7.0 to call breakpoint genotypes of the structural variants (Chiang, et al. 2015)â€.
- [l.512â€“513] **S:** The sentence did not make sense. I suggest to rephrase to â€œThe script we used for CNV calling, merging, and breakpoint genotypic is available from <URL>â€. (Note that the URL leads to a GitLab repository two which external users have no access.)
- [l.516â€“517] This sentence in parts seems to be a verbatim copy of the description on the tool's website. To avoid the risk of committing plagiarism, I recommend to rephrase the description. I think the copying of the description from the website also transferred a statement from the website that I found misleading: as far as I understand, what CNV-Sim does is to first simulate NGS reads using ART, and then to introduce CNV. In sum, I think that l.515 to 519 could be simplified to and written more clearly as â€œTo assess the sensitivity and accuracy of our pipeline for CNV calling, we simulated short-read data with CNV using CNV-Sim v0.9.2 (https://github.com/NabaviLab/CNV-Sim)â€. Since you will ultimately provide the script, interested readers can look up the details there.
- [l.521] **S:** Rephrase â€œThe command lines for simulations can be found on: ...â€ to â€œThe script implementing these CNV simulations is available from ...â€. (Again, note that external users have no access to the linked GitLab repository. Please make the repository public.)
- [l.535â€“537] **R:** Please consistently do or do not surround the "equals" symbols ("=") by spaces.
- [l.560] **S:** Omit â€œthenâ€ before â€œindependentlyâ€. Omit â€œvalueâ€ after â€œ$V_{\mathrm{ST}}$â€.
- [l.562] **C:** The linked GitLab repository is not freely accessible. Please make the repository public.
- [l.573] **S:** Insert â€œaâ€ before â€œBLASTâ€.
- [l.575] **S:** Insert â€œaâ€ before â€œGO enrichment analysisâ€ and â€œtheâ€ before â€œ*A. thaliana*â€.
- [l.576â€“578] **R:** This sentence reads unnecessarily complicated. Please rephrase to something like â€œWe used the Benjamini-Hochberg method (reference) to control the false discovery rate at 0.5 when we determined the enriched GO termsâ€.
- [l.585â€“586] **S:** Replace â€œwe performed analysis of gene CN expansion and contractionâ€ by â€œwe analysed gene CN expansion and contractionâ€.
- [l.591â€“592] **R:** This sentence does not read well. Please adjust to something like â€œWe chose a significance threshold of 0.05 when identifying genes with an excess rate of evolution (expansion or contraction) in different groups/populationsâ€.
- [l.592] **S:** Rephrase to â€œThe code we used to analyse CN expansion and contraction can be found on ...â€. (Note that the linked GitLab repository cannot be freely accessed.)
- [l.600] **S:** Please insert â€œtheâ€ before RDA analysis. Replace â€œfrom the vegan package as implemented in Râ€ by â€œfrom the R package veganâ€.
- [l.604] **S:** Replace â€œthat showed VIF > 10â€ by â€œwith a VIF of 10 or aboveâ€. Insert â€œtheâ€ before â€œRDAâ€.
- [l.605] **S:** Replace â€œThe R script of RDA analysesâ€ by â€œThe R script for the RDA analysisâ€. (Please note that the URL leads to a GitLab repository that is not freely accessible.)
- [l.607â€“608] **C/S:** I did not understand what exactly the authors mean by â€œthe dynamics of CN across populationsâ€. Should there be a â€œtheâ€ before â€œRDAâ€?
- [l.608] **C/S:** The authors so far used â€œclimatic variable(s)â€, but here they use â€œclimate variableâ€. I suggest the authors unify their use of language.
-  **TODO(Simon): Go on here.** (l.610)

### Figures
- ...
- Fig. 1:
    - **C:** ...
- Fig. 2:
    - **C:** ...
- Fig. 4:
    - **C:** ...
    - [l.pâ€“p] **C:** ...
- Fig. 5:
    - [l.pâ€“p] **C:** ...

### Tables
- Table 1: ...

### Supplementary Text
- ...

### Supplementary Figures and Tables
- **TODO(Simon): Check if the notes below are old or current.**
- Figure S2:
    - *There seems to be again a confusion between â€œaccessionâ€ and â€œindividualâ€. In the label of the x-axis, it should say â€œindividualsâ€œ in my view.*
- Figure S3:
    - **R:** See my comment to Fig. 2 w.r.t. the colour scheme of the ADMIXTURE plot in panel C (it does not match the colours assigned to the accessions and used in panel A).
- Figure S5:
    - **Q/R:** *Do the dots in the figure represent genomic windows or genes? Please adjust the caption if necessary (it currently mentions â€œgenesâ€).*
- Figure S6:
    - *What is meant by â€œCN value(s)â€? Please define / write out once at least.*
- Figure S7:
    - ***C/R**: The description in the caption of panel C is unclear. Do you mean Do you mean â€œThe number of genes associated with a response to ...â€? Please fix.*
- Figure S9:
    - ***Q/R:** Am I right in thinking that this figure is reproduced from Wei et al. (2023a)? If so, I do not think it is necessary (nor appropriate) to reproduce the figure here.*
- Figure S11:
    - **C/R:** *The caption misses a title sentence. Please fix.*
    - **R:** *The second part of the caption needs revision. The wording used to describe the RDA plot is unclear. See detailed comments in the annotated PDF.*
- Figure S12:
    - **R:** *Please add a title sentence to the caption and describe what panel A shows.*
- Figure S13:
    - **R:** *Please revise the title sentence to be more informative.*
- Table S3:
    - **R:** *The table caption does not inform about the meaning of the numbers given outside and inside of the parentheses. One may suspect that the numbers report the true and false positives, but this remains unclear. Please describe what the numbers mean.*
- Table S4:
    - **C:** *This table seems fully redundant to Fig. 1E and could in my view be omitted.*
- Table S5:
    - **C:** *I do not think the footnote is necessary given the column labels in the table and the details given in the Methods and Supplementary Text.*

    
## Notes
- It is unclear how $V_{\mathrm{ST}}$ works.