---
title: Review of Manuscript MBE-24-1101 by Wei *et al.* “Copy number variations shape genomic structural diversity assiciated with ecological adaptation in the wild tomato *Solanum chilense*”
author: Simon Aeschbacher
date: 22 November 2024
geometry: "left=3cm,right=3cm,top=2.5cm,bottom=2.5cm"
output: pdf_document
linestretch: 1.15
---

## Summary
Wei et al. explore the the extent, nature, evolution, and adaptive role of copy number variation (CNV) in the wild tomato species *S. chilense* using whole-genome re-sequencing data derived from seven populations from three geographic areas representing three contrasting environments. I consider the purpose of this study highly relevant and of high interest.
The authors detected 212,207 in an ensemble approach based on four different software tools. The authors validated this approach using simulated data and showing that their approach reduced the false positive rate relative to any single calling tool. The authors analysed population differentiation from CNV using PCA and ADMIXTURE to find that the southern coastal populations are strongly differentiated from the other two groups (central, southern highland), and strongly differentiated amongst each other. The authors interpret the concordance of these two results as indicating that CNV at large is driven by demographic history and recent colonisation of the two southern areas.
To explore how CNV varies among populations and might be related to gene function, the authors first identified genes with particularly high differentiation in copy number among the populations using a relative measure of between-population variation analogous to $F_{\mathrm{ST}}$, and then performed a gene ontology enrichment analysis for the highly CN-differentiated genes. The authors identified 3,539 (2,192) genes (very) highly differentiated in CN among the populations. The southern highland and southern coastal populations differed in the relative extents of duplication vs. deletion at these genes. The authors fond the CN-differentiated genes to be functionally associated with abiotic stress (drought, cold, heat, and light) and pathways involved in the regulation of flowering time (sensitivity to photoperiod, vernalisation). The authors take these results as evidence that selective pressures linked to divergent habitat is manifested in CNV, and that adaptive changes in CN might have facilitated the colonisation of extreme habitats in the south of the contemporary species range. The authors also explored the dynamics of CN evolution and found overall trends of CN contraction in central and southern coastal populations, and CN expansion in southern highland populations.
To further explore potential evidence for adaptive divergence at CNV, the authors performed genome-environment association (GEA) analyses using redundancy analysis to identify a set of climate variables associated with CN variation, followed by latent factor mixed modelling to identify gene sets associated with individual climate variables. The GEA revealed six climate variables representing variation in temperature, solar radiation, and potential evapotranspiration to be highly associated with CN differentiation. The authors identified 312 CN-differentiated genes highly associated with these six climatic variables. A subset of 34 CN-differentiated genes was found to be associated with both annual temperature range as well as annual mean solar radiation, and patterns of CN divergence among the populations at these genes suggested an overrepresentation of duplications in populations at high elevations (one in the center, two in the southern part of the species range).
The authors conclude that the patterns of CNV they found among the populations studied, as well as the inferred dynamics of CN (expansion, contraction), are driven both by the demographic history (spatial expansion and bottlenecks) as well as spatially divergent selection promoting local adaptation to the different habitats. The authors therefore suggest that genomic studies of adaptive divergence in natural populations should take into account structural genomic variation as a potential source of causal or linked variation informative about ecological adaptation.
I found this manuscript to be of high scientific relevance. The methods and approaches seem to have been carefully chosen and well implemented. I detected no major flaws. However, the authors seem to be overly confident in an adaptive evolutionary explanation of the observed patterns of CNV. I suggest the authors change their wording to be more descriptive and more balanced at some places in the text (see Minor Comments). Unfortunately, the manuscript suffers from many issues with language and writing that should have been addressed before a first submission. These issues together make for a major issue. For this reason, and in spite of the scientific quality, I need to suggest major revisions.

## Major Comments
1. ...
2. I was not able to access the script files from the GitLab repository at the LRZ center for computation. I therefore could not (spot) check the code. I consider this a limitation that should be fixed in the process of peer review if the journal wants the review to cover the supplementary code files.

## Minor Comments
**C:** comment; **Q:** question; **S:** suggestion; **R:** request.

### Title
- **C:** I appreciate the modification from “underpinning” to “associated”.

### Abstract
- ...

### Introduction
- [l.p–p] **C:** 

### Results
- [l.p–p] **C:**

### Discussion
- [l.p–p] **C:**

### Materials and Methods
- [l.492–492] **C:** The abbreviation “SV” for “structural variation” has already been introduced in the main text. The repeated definition here likely results from the fact that the authors moved some methods text from the Supporting Materials to the main text.
- [l.493–495] **S:** Perhaps rephrase to “This evaluation enumerated ... and/or duplications.” to “This study found that combining SV detection tools tends to give higher precision and recall than individual tools and that LUMPY (Layer, et al. 2014), Manta (Chen, et al. 2016), Wham (Kronenberg, et al. 2015) and DELLY (Rausch, et al. 2012) are tools with good overall performance for deletions and duplications.”
- [l.495–497] **C:** This sentence does not seem to be complete. It remains unclear what the difference is between “tool” and “algorithm”. It seems as if the authors use “algorithm” for what Kosugi et al. (2019) used “method”, whereas Kosugi et al. (2019) used “algorithm” for the software tool. **S:** I suggest that the authors restrict their statement here to what is strictly necessary for the reader to obtain an overview of what information from the sequencing data data was used to call CV overall. Later in the text, when the application of specific tools is described, the description can state what information the respective tool relies on. Specific suggestion for here: “These tools implement different calling algorithm that jointly draw information from patterns in read pairs, split reads, read depth, and de novo assembly”.
- [l.498] **S:** For consistency with l. 493, I suggest to write “LUMPY” in capital letters. This comment also applies to other occurrences of the tool name.
- [l.498–500] **S:** Rephrase “, and the split-read alignments also were extracted ...” to “, and then we extracted the split-read alignments ...” to increase consistency (active voice) and clarity (repetition of the phrasing reassures to the reader that both steps are done to extract data to be analysed, not to be excluded).
- [l.502] **S:** Rephrase “CNV calling used DELLY ... into vcf file (Danecek, et al. 2011; Danecek, et al. 2021)” to “For CNV calling with DELLY v0.7.6, we chose the default parameters and converted the output file from bcf to vcf format using bcftools v1.9 (Danecek, et al. 2011; Danecek, et al. 2021)”.
- [l.504] **S:** Omit “Furthermore” and just say “We also ran ...”.
- [l.506–507] **S:** Replace “as 50bp” and “as 1Mb” by “to 50bp” and “to 1 Mb”, respectively. **C:** It was unclear to me what is meant by “..., and CNV types and DNA strands must match“. Please clarify and improve the formulation.
- [l.509] **R:** The current phrasing does not work. Please rephrase to “We (finally) used the merged CNV set **as input** for SVTyper v0.7.0 to ...”.
- [l.509–512] **S:** I think this part could be written more compactly as “We used the merged CNV set as input for SVTyper v0.7.0 to call breakpoint genotypes of the structural variants (Chiang, et al. 2015)”.
- [l.512–513] **S:** The sentence did not make sense. I suggest to rephrase to “The script we used for CNV calling, merging, and breakpoint genotypic is available from <URL>”.
- [l.516–517] This sentence in parts seems to be a verbatim copy of the description on the tool's website. To avoid the risk of committing plagiarism, I recommend to rephrase the description. I think the copying of the description from the website also transferred a statement from the website that I found misleading: as far as I understand, what CNV-Sim does is to first simulate NGS reads using ART, and then to then introduce CNV. In sum, I think that l.515 to 519 could be simplified to and written more clearly as “To assess the sensitivity and accuracy of our pipeline for CNV calling, we simulated short-read data with CNV using CNV-Sim v0.9.2 (https://github.com/NabaviLab/CNV-Sim)”. Since you will ultimately provide the script, interested readers can look up the details there.
- [l.521] **S:** Rephrase “The command lines for simulations can be found on: ...” to “The script implementing these CNV simulations is available from ...”.

### Figures
- ...
- Fig. 1:
    - **C:** ...
- Fig. 2:
    - **C:** ...
- Fig. 4:
    - **C:** ...
    - [l.pp–pp] **C:** ...
- Fig. 5:
    - [l.p–p] **C:** ...

### Tables
- Table 1: ...

### Supplementary Text
- Please refer to the annotated PDF. I did not make detailed suggestions for the entire text because I was a bit overwhelmed by the density of writing and language issues. Please revise the entire text carefully before submitting a revision.

### Supplementary Figures and Tables
- Generic comment: The captions of the supplementary figures and tables seem incomplete and rudimentary at times. Please add full title sentences and also make full sentences in the remainder of the captions.
- Figure S2:
    - There seems to be again a confusion between “accession” and “individual”. In the label of the x-axis, it should say “individuals“ in my view.
- Figure S3:
    - **R:** See my comment to Fig. 2 w.r.t. the colour scheme of the ADMIXTURE plot in panel C (it does not match the colours assigned to the accessions and used in panel A).
- Figure S5:
    - **Q/R:** Do the dots in the figure represent genomic windows or genes? Please adjust the caption if necessary (it currently mentions “genes”).
- Figure S6:
    - What is meant by “CN value(s)”? Please define / write out once at least.
- Figure S7:
    - **C/R**: The description in the caption of panel C is unclear. Do you mean Do you mean “The number of genes associated with a response to ...”? Please fix.
- Figure S9:
    - **Q/R:** Am I right in thinking that this figure is reproduced from Wei et al. (2023a)? If so, I do not think it is necessary (nor appropriate) to reproduce the figure here.
- Figure S11:
    - **C/R:** The caption misses a title sentence. Please fix.
    - **R:** The second part of the caption needs revision. The wording used to describe the RDA plot is unclear. See detailed comments in the annotated PDF.
- Figure S12:
    - **R:** Please add a title sentence to the caption and describe what panel A shows.
- Figure S13:
    - **R:** Please revise the title sentence to be more informative.
- Table S3:
    - **C:** This table seems fully redundant to Figure S2 and could in my view be omitted.
- Table S4:
    - **C:** This table seems fully redundant to Fig. 1E and could in my view be omitted.
- Table S5:
    - **C:** I do not think the footnote is necessary given the column labels in the table and the details given in the Methods and Supplementary Text.